tests/tests/test_deployment_aborting.py

```diff
@@ -94,12 +97,21 @@ def test_deployment_abortion_instantly(
     @MenderTesting.fast
     def test_deployment_abortion_downloading(
         self, standard_setup_one_client_bootstrapped, valid_image
+    ):
+        self.abort_deployment(
+            standard_setup_one_client_bootstrapped, valid_image, "downloading",
```

Comment:
Here and above we have an opportunity to make the test more readable by explicitly stating the parameters:
```suggestion
            standard_setup_one_client_bootstrapped, valid_image, "downloading",
            mender_performs_reboot=False
```

(or, if we do the suggestion above, `number_of_reboots=0`)

In general I think in test code the more explicit and less default values the more readable. Don't you agree?